app/models/manageiq/providers/azure/cloud_manager/refresh_parser.rb

```diff
@@ -649,7 +649,7 @@ def parse_image(image)
           :publicly_available => false,
           :hardware           => {
             :bitness  => 64,
-            :guest_os => image.operating_system
+            :guest_os => OperatingSystem.normalize_os_name(image.operating_system)
```

Comment:
@Ladas - slipping this little change into the old parser?

Reply:
Right, I have specs comparing old and new refresh. For some reason old refresh has guest_os normalized, except for this kind of image.